src/node_buffer.cc

```diff
@@ -282,7 +282,13 @@ MaybeLocal<Object> New(Isolate* isolate,
       free(data);
       data = nullptr;
     } else if (actual < length) {
-      data = static_cast<char*>(realloc(data, actual));
+      char* new_data = static_cast<char*>(realloc(data, actual));
+      if (new_data == NULL) {
+        data = nullptr;
```

Comment:
Ah, you're right. I can just change it to `if (new_data != nullptr) data = new_data;`